releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -6,6 +6,14 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 1.1.5/1.2.5 - April 27, 2020
+
+* Added support to allow users to configure the client used to connect to App Configuration and Key Vault. Users can now define the clients via `ConfigurationClientBuilderSetup` and `SecretClientBuilderSetup`, shown [here](https://github.com/microsoft/spring-cloud-azure/tree/master/spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config#modifying-connection-client).
+  * This allows for setting up additional client information such as proxy information. [#656](https://github.com/microsoft/spring-cloud-azure/issues/656)
+* Fixed the bug that caused partial configuration to load when both `/application/` and `/<application_name>/` are used, and only one of them loaded. This also caused the store to be unable to refresh.
+* Improved the performance of refresh by reducing the number of requests made to App Configuration during configuration change detection. [#672](https://github.com/microsoft/spring-cloud-azure/issues/672)
+* Fixed the bug where using an empty string label "\0" or a comma without a value before it could not be used. [#655](https://github.com/microsoft/spring-cloud-azure/issues/655)
```

Comment:
The "%00" was working. Not sure when it was broken. Using "\0" as null label is a breaking change... at least from a documentation perspective. Suggest below.

* Fixed the bug that `%00` was not working when it's used to indicate keys with no labels explicitly. Changed to use `\0` to indicate keys with no labels. For example, a property setting below means loading keys with no labels and then overwritten by keys with label `dev`. 
    ```properties
    spring.cloud.azure.appconfiguration.stores[0].label = \0,dev
    ```

Reply:
I looked into it and it looks like there was a slight change the label check when it switched to using the Azure sdk.

Reply:
Yes, I suspect this was broken during that time too.